solr/solr-ref-guide/src/collection-management.adoc

```diff
@@ -1250,32 +1250,7 @@ http://localhost:8983/solr/admin/collections?action=COLSTATUS&collection=getting
                                     "_i_Lucene50_0.pos": "11.1 MB",
                                     "_i_Lucene50_0.doc": "10 MB",
                                     "_i_Lucene50_0.tim": "4.3 MB"
-                                },
-                                "ramBytesUsed": {
```

Comment:
I confirmed with a local test that this section is now gone in COLSTATUS output

Reply:
Looks good to me.